app/actions/remote/user.ts

```diff
@@ -92,7 +93,12 @@ export const fetchProfilesPerChannels = async (serverUrl: string, channelIds: st
     }
 };
 
-export const loadMe = async (serverUrl: string, {deviceToken, user}: LoadMeArgs) => {
+export const loadMe = async (serverUrl: string, {deviceToken, user}: LoadMeArgs): Promise<{
```

Comment:
definitely thinking of removing this function or we would need to adapt it in order for us to do a proper sync